eng/native/ijw/IJW.cmake

```diff
@@ -65,8 +65,7 @@ if (CLR_CMAKE_HOST_WIN32)
   set_directory_properties(PROPERTIES COMPILE_OPTIONS "${dirCompileOptions}")
 
   # IJW tests needs to load DLLs from somewhere other than System32
-  string(REPLACE "/DEPENDENTLOADFLAG:0x800" "" CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS}")
-  string(REPLACE "/DEPENDENTLOADFLAG:0x800" "" CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS}")
+  string(REPLACE "/DEPENDENTLOADFLAG:0x800" "" LINK_OPTIONS "${LINK_OPTIONS}")
```

Comment:
You probably need to use `get_directory_property` and `set_directory_properties` here to get this to work right.